app/models/digital_object.rb

```diff
@@ -17,10 +17,9 @@ class DigitalObject < ApplicationRecord
   include DigitalObjectConcerns::ParentChildBehavior
   include DigitalObjectConcerns::ResourceImports
 
-  # Special structure for the title dynamic field
-  TITLE_DYNAMIC_FIELD_GROUP_NAME = 'title'
-  TITLE_SORT_PORTION_DYNAMIC_FIELD_NAME = 'sort_portion'
-  TITLE_NON_SORT_PORTION_DYNAMIC_FIELD_NAME = 'non_sort_portion'
+  # Access keys for the title attribute
+  TITLE_SORT_PORTION_KEY = 'sort_portion'
+  TITLE_NON_SORT_PORTION_KEY = 'non_sort_portion'
```

Comment:
It seems like these constants aren't doing a lot for us.  They're only used in the generate_label method below.  Maybe it's worth getting rid of them?  Or if it's useful to have globally-accessible constants like these we should put them in the title TypeDef class?

Also, `'value'` is a hard-coded string in the `generate_label` method, so if these keys are treated as special constants then maybe that should be too.

But I'm also fine with just getting rid of the constants and using strings in the generate_label method if that simplifies things.

Reply:
Inlined!